package.json

```diff
@@ -37,30 +37,22 @@
   },
   "devDependencies": {
     "babel-cli": "^6.18.0",
-    "babel-core": "^6.18.0",
-    "babel-jest": "^16.0.0",
-    "babel-plugin-transform-export-extensions": "^6.8.0",
-    "babel-polyfill": "^6.16.0",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-stage-2": "^6.18.0",
+    "babel-core": "^6.21.0",
+    "babel-jest": "^18.0.0",
+    "babel-polyfill": "^6.20.0",
+    "babel-preset-env": "^1.1.5",
     "babel-register": "^6.18.0",
-    "eslint": "^3.9.0",
-    "eslint-config-airbnb-base": "^9.0.0",
-    "eslint-plugin-babel": "^3.3.0",
-    "eslint-plugin-import": "^2.0.1",
+    "eslint": "^3.12.2",
+    "eslint-config-airbnb-base": "^11.0.0",
+    "eslint-plugin-babel": "^4.0.0",
+    "eslint-plugin-import": "^2.2.0",
     "flux-standard-action": "^1.0.0",
-    "jest": "^16.0.2",
+    "jest-cli": "^18.1.0",
     "redux": "^3.6.0",
     "redux-mock-store": "^1.2.1",
     "rimraf": "^2.5.4"
   },
   "dependencies": {
     "sanctuary": "^0.11.1"
-  },
-  "babel": {
-    "presets": [
-      "es2015",
-      "stage-2"
-    ]
```

Comment:
If this, and the `.babelrc`s reference to `presets` are going away, where is babel pulling its config from?